packages/widget_toolkit/lib/src/lib_edit_address/ui_components/edit_address_form.dart

```diff
@@ -71,7 +60,8 @@ class EditAddressForm<T extends PickerItemModel> extends StatelessWidget {
                       bloc.events.saveCountry();
                     },
                     modalConfiguration: countryPickerModalConfiguration,
-                    searchCountryService: searchCountryService,
+                    searchCountryService:
+                        CountryService(editAddressService: editAddressService),
```

Comment:
Please avoid creating instances of CountryService on each rebuild

Reply:
I have moved the creation of the services; CityService, StreetService and CountryService in the edit_address_dependencies.dart file